### PR TITLE
INFRA Initialize Acceptance tests 

### LINKED ISSUE #3: INFRA: Initialize Acceptance Test Project

### PR DESCRIPTION
**Pull request by Manasa1191**
Closes #3

### CODE REVIEW

**Comment by Pwongchaiya** [comment]
Add "Close: #3" where it says "No description is provided"

**Comment by hassanhabib** [comment]
@Manasa1191 fix the name of your PR

**Comment by Manasa1191** [comment]
Will do it by EOD. Thank you

File: InternTrack.Core.Api.Tests.Acceptance/InternTrack.Core.Api.Tests.Acceptance.csproj [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```
Comment (hassanhabib) [inline InternTrack.Core.Api.Tests.Acceptance/InternTrack.Core.Api.Tests.Acceptance.csproj:5]: both needs to be disabled this one and the one below it.

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+{
+    public partial class HomeApiTests
+    {
+        [Fact]
+        public async Task ShouldReturnHomeMessageAsync()
+        {
+            // Given
+            string expectedMessage = 
+                "Thank you, Mario. But the princess is in another castle!";
+            // When
```
Comment (hassanhabib) [inline InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs:17]: New line before `// when` and make sure you lower case all letters `// given` and `// when` and `// then`

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+
+using FluentAssertions;
+using Xunit;
+namespace InternTrack.Core.Api.Tests.Acceptance.Apis.Home
```
Comment (hassanhabib) [inline InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs:7]: New line above `namespace ...`

File: InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs (hunk 2 above)
Comment (hassanhabib) [reply InternTrack.Core.Api.Tests.Acceptance/Apis/Home/HomeApiTests.Logic.cs:17]: Do the same for `// then` need new line

File: InternTrack.Core.Api.Tests.Acceptance/InternTrack.Core.Api.Tests.Acceptance.csproj (hunk 1 above)
Comment (Manasa1191) [reply InternTrack.Core.Api.Tests.Acceptance/InternTrack.Core.Api.Tests.Acceptance.csproj:5]: changed to disable. But build fails. 
Screenshot (48)

**Pull request closed without merging**